.rubocop.yml

```diff
@@ -1,21 +1,62 @@
-inherit_from: .rubocop_todo.yml
 AllCops:
   Exclude:
+    - 'tmp/**/*'
     - 'vendor/**/*'
-    - 'db/**/*'
-    - 'config/**/*'
-    - 'script/**/*'
-    - 'spec/**/*'
-    # Excluded because it's in such a bad state
-    - 'lib/supplejack/controllers/helpers.rb'
-    - 'lib/supplejack/config.rb'
   DisplayStyleGuide: true
-  # prevent bug caused by https://github.com/bbatsov/rubocop/issues/2213 on travis
-  # TODO remove
-  UseCache: false
+  DisplayCopNames: true
+Rails:
+  Enabled: true
+# https://github.com/bbatsov/rubocop/issues/1633
 Metrics/LineLength:
-  Max: 120
+  Enabled: false
+Style/Documentation:
+  Enabled: false
+Metrics/PerceivedComplexity:
+  Enabled: false
+Metrics/CyclomaticComplexity:
+  Enabled: false
+Metrics/ParameterLists:
+  Enabled: false
 Metrics/MethodLength:
-  Max: 25
-Style/MultilineOperationIndentation:
   Enabled: false
+Metrics/AbcSize:
```

Comment:
Instead of disabling all of these cops you can create a rubocop todo file which will turn off all of the cops for existing code and put the offenses in a yaml file. That way we can make sure new code is copped but old code only gets copped when it is updated.

Reply:
Cool, added a change for that